apps/resource_manager/lib/identities/commands/inputs/create_user.ex

```diff
@@ -5,40 +5,63 @@ defmodule ResourceManager.Identities.Commands.Inputs.CreateUser do
 
   use ResourceManager.Input
 
+  alias ResourceManager.Credentials.Schemas.Password
   alias ResourceManager.Identities.Schemas.User
 
   @typedoc "Create user input fields"
   @type t :: %__MODULE__{
           username: String.t(),
           status: String.t(),
-          password_hash: String.t(),
-          password_algorithm: String.t(),
-          scopes: list(String.t()) | nil
+          password: %{
+            value: String.t(),
```

Comment:
```suggestion
            hash: String.t(),
```

Reply:
We may don't want to lose the explicit information that the value stored is hashed, not plain

Reply:
In this case it's the play password that we receive and not the hashed  one.